app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -10,106 +10,105 @@
       name="isListExpanded"
       type="Boolean" />
 
-     <variable
+    <variable
       name="viewModel"
       type="org.oppia.android.app.hintsandsolution.HintsViewModel" />
   </data>
 
-    <LinearLayout
-      android:id="@+id/hint_summary_container"
+  <LinearLayout
```

Comment:
We're aiming to flatten the view hierarchy by replacing the linear layout here by constraint layout. Try approaching that way.

Reply:
Thank you for the review @Sarthak2601. 
Do you mean every Linear layout should be replaced with constraint layout?

Reply:
We're trying to get the most optimised UI here. So try to flatten the view hierarchy as much as you can without disrupting the existing design. That being said, if you think you can change both-give it a shot and if things go south, at least replace the outermost linear layout.

Reply:
Ok, I think I understand now. It is just fir this layer

Reply:
Done